.github/workflows/auto-assign.yml

```
@@ -0,0 +1,20 @@
+name: Pull request and issue auto-assign
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requestS: write
```

actionlint (1.7.10)

unknown permission scope "pull-requestS". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)